.dockerignore

```diff
@@ -0,0 +1,6 @@
+Dockerfile
+.env
+.gitignore
+.venv
+.vscode
+ansible
```

Comment:
Nice - you've got the most important files to ignore here, so even though your copy shouldn't hit these you're still fairly safe.
That said, and possibly something you're already conscious of, but it's worth noting that [.dockerignore & .gitignore use slightly different syntaxes that can catch you out](https://zzz.buzz/2018/05/23/differences-of-rules-between-gitignore-and-dockerignore/) - a key difference for example being that if someone placed the .env file _inside_ the todo_app folder, that would still get copied in currently! If we want to avoid that, we'd need to switch to use `**/.env`